setup.py

```diff
@@ -58,8 +56,7 @@
         'nose',
         'httpretty',
         'mock',
-        'factory_boy',
-        ndg_httpsclient_version
```

Comment:
no longer needed because of this: https://github.com/quandl/quandl-python/blob/AP-1624/backwards-compatibility/test/helpers/my_httpretty.py , the version lock to 0.3.0 was only for testing purposes